doc/api/async_hooks.md

```diff
@@ -758,6 +758,16 @@ const server = net.createServer((conn) => {
 Promise contexts may not get valid `triggerAsyncId`s by default. See
 the section on [promise execution tracking][].
 
+### `async_hooks.asyncWrapProviders`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: A list of providers supported by async wrap.
```

Comment:
Do we define anywhere what a "provider" is?
Also, to me "A list" means an array, but an object is returned.

Reply:
I think that it wasn't defined, at least publically. I can reword that for sure, but, to be honest, I don't know how to define the `providers` description.

Reply:
```suggestion
* Returns: A map of provider types supported by async wrap.
```

I would also add something to the description linking it to `type` parameter of the `init(...)` hook.

Reply:
Just did, thanks for point that out!